### ISSUE TITLE
Search Result page

### ISSUE THREAD

**Issue by sattastudd** [sattastudd/KhanaShana#8]
1. Values to be post in 3 forms
	a) Dish name
	b) Cuisines
	c) Locality
   The result will be according to any one of the above query.

2. Result values to be chanegd according to filter applied.
3. Incoming values

       	a) Restuarant Name
	b) Rating
	c) Minimum Delivery

**Comment by sattastudd** [comment]
Fields for Avg cost, Delivery time, min delivery, open hours timing need to added at admin side.
while adding new restaurants.

**Comment by grvtiwari** [comment]
Noted. 
Where will be these values ? 

Should not these values be entered when the restaurant is in final approval queue ?

**Comment by sattastudd** [comment]
Yes these values should be inserted while adding new restaurant.

And what th fuck u were doing at 5 in the morning :p

Bhai 7 ghante ki need bahut Zamora hai.
